lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html

```diff
@@ -74,15 +65,14 @@
             [floatLabel]="'never'">
             <input matInput
                 class="adf-property-value"
+                [ngClass]="{'adf-property-value-editable': editable}"
```

Comment:
I don't see this class in scss file for this component. Is not that missing?

Reply:
reusable class in the host component